public/app/plugins/datasource/grafana-azure-monitor-datasource/components/VariableEditor/VariableEditor.tsx

```diff
@@ -286,6 +288,26 @@ const VariableEditor = (props: Props) => {
           />
         </InlineField>
       )}
+      {query.queryType === AzureQueryType.AzureResourceGraph && (
+        <>
+          <ArgQueryEditor
+            subscriptionId={query.subscription}
+            query={query}
+            datasource={datasource}
+            onChange={onQueryChange}
+            variableOptionGroup={variableOptionGroup}
+            setError={setError}
+          />
+          {errorMessage && (
+            <>
+              <Space v={2} />
+              <Alert severity="error" title="An error occurred while requesting data from Azure Monitor">
+                {errorMessage}
+              </Alert>
+            </>
+          )}
```

Comment:
I tried to trigger this, writing an invalid query, but I am not able to see the error in the UI, just an uncaught error in the console:

![Screenshot from 2022-08-02 16-47-26](https://user-images.githubusercontent.com/4025665/182403867-1b89d7a7-df97-4599-871c-b18491ad3ee4.png)

It should show an error like the other variable types, this is for example if I introduce a wrong subscription when requesting resource groups:

![Screenshot from 2022-08-02 16-49-46](https://user-images.githubusercontent.com/4025665/182404316-12732054-c0cd-4de0-aa12-acc0d5100f00.png)